rubix_admin/installer.py

```diff
@@ -79,19 +83,11 @@ def _rpm_install(cls, args):
                               os.path.basename(rpm))))
 
     @classmethod
-    def _rubix_op(cls, args):
-        sudo("cp -a /usr/lib/rubix/lib/* /usr/lib/presto/plugin/hive-hadoop2/")
-        sudo("cp -a /usr/lib/rubix/lib/* /usr/lib/hadoop/lib/")
-        sudo("mkdir -p /mnt/rubix/")
-        sudo("mkdir -p /var/lib/rubix/cache")
-        with settings(warn_only=True):
-          sudo("ln -s /var/lib/rubix/cache /mnt/rubix/")
-        count = 0;
-        while count < 5:
-          sudo("mkdir -p /var/lib/rubix/cache/data%s" % count)
-          count += 1
-        sudo("mkdir -p /etc/presto/conf/catalog")
+    def _rubix_op(cls, args, is_master):
+        sudo("chmod +x /usr/lib/rubix/bin/configure-*.sh")
+        sudo("/usr/lib/rubix/bin/configure-rubix.sh")
 
-        sudo("chmod +x /usr/lib/rubix/bin/configure.sh")
-        sudo("/usr/lib/rubix/bin/configure.sh")
-        sudo("restart presto-server")
+        cluster_type = args.cluster_type
+        if not (is_master is False and cluster_type == "spark"):
```

Comment:
Can you change this to separate spark and presto ?

Reply:
Updated to improve clarity